addons/rose/addon/components/rose/anonymous/index.js

```diff
@@ -3,6 +3,12 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+/* eslint-disable */
+// TODO: Convert this component into a glimmer component.
+// Currently, this file causes three eslint errors.
+// 1. Use Glimmer components instead of classic components.
+// 2. Native JS classes should be used instead of classic classes.
+// 3. Require tagless components by converting to a Glimmer component.
```

Comment:
Disabling eslint for now since, `rose/anonymous` is used in various places with the `tagName` attribute to dynamically set the html tag to be used. So this refactor should be separated out in a diff. PR.

Reply:
None blocking comment: this is a interesting refactor due to the usage we give to this "component". Do we have a jira task to cover this work? or the plan is to bring it first to tech time for debate? BTW very nice one, this one is not goin to be straigforward 😅  good eye!

Reply:
oh yeah im planning to bring it up as techtime topic (i'll update the doc rn).